src/style.css

```diff
@@ -0,0 +1,42 @@
+* {
```

Comment:
- [ ] You should style your app to make it look like the [wireframe ](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png )provided, please do so to comply with the project requirements.